app/code/community/Nosto/Tagging/Model/Meta/Product/Trait.php

```diff
@@ -112,6 +112,11 @@ protected function buildProductPrice(Mage_Catalog_Model_Product $product, Mage_C
         /** @var Nosto_Tagging_Helper_Price $priceHelper */
         $priceHelper = Mage::helper('nosto_tagging/price');
 
+        /** @var Nosto_Tagging_Helper_Data $dataHelper */
+        $dataHelper = Mage::helper('nosto_tagging/data');
+        if ($dataHelper->isVariationEnabled()) {
```

Comment:
We must pass the store here

```suggestion
        if ($dataHelper->isVariationEnabled($store)) {
```

Reply:
Fixed, thanks